### DISCUSSION TITLE
Upload Zen Browser to MS Store, Scoop and other repositories

### DISCUSSION THREAD

**Discussion by Bammede** [zen-browser/desktop#1303] [Ideas]
Upload Zen Browser to MS Store, Scoop and other repositories

**Comment by anxshi** [comment]
+1 for Scoop

**Comment by EFLKumo** [comment]
+1 for scoop

**Comment by tedbergstrand** [comment]
I disagree. This bad boy's not ready for prime-time, yet. (Don't get me wrong, I love Zen Browser and I'm super appreciative to the devs.) But there are too many changes and they're causing too many bugs. And I think it would be a bad idea to blast this out so widely at this point.

You don't usually get a second chance at a first impression with a browser. I think it should be more stable before it goes out on other platforms for wider distribution. Frankly, I've had to roll back updates to a week ago to ensure a stable browsing experience. That's not something regular users are interested in futzing with.

I think it's important to remember that most of us are fairly technically inclined and understand the realities of being an early adopter. Microsoft Store users generally don't think the same way.

**Comment by universish** [comment]
There's no need. This requires additional costs and additional human resources. You can. This is an open source project. If you are willing to solve bugs, join the community and take on this task and responsibility. If you want it so bad, go ahead and do it.

**Comment by JaHIY** [comment]
+1 for Scoop
